frontend/templates/layouts/mail.html

```diff
@@ -0,0 +1,11 @@
+<html>
+  <head></head>
+  <body>
+    <p>Salaam {{username}},</p>
+    <p>Thank you for submitting your corrections to Sunnah.com</p>
+    <p>{{message}}</p>
```

Comment:
Instead of this being message. Let us mention the hadith that they submitted the correction for and the kind of correction they made. You can check the correction object to figure out what we can use to populate the template

Reply:
Template body has been updated